commons/ava_testsuite/rpc_manager.go

```diff
@@ -0,0 +1,147 @@
+package ava_testsuite
+
+import (
+	"github.com/kurtosis-tech/ava-e2e-tests/commons/ava_default_testnet"
+	"github.com/kurtosis-tech/ava-e2e-tests/gecko_client"
+	"github.com/palantir/stacktrace"
+	"github.com/sirupsen/logrus"
+	"time"
+)
+
+const (
+	GENESIS_USERNAME = "genesis"
+	GENESIS_PASSWORD = "genesis34!23"
+	ACCEPTED_STATUS = "Accepted"
+)
+
+type RpcManager struct {
```

Comment:
Can we be more specific about what this guy does? It _seems_ like what he's really doing is bootstrapping a PChain account with Ava

Reply:
this.basically provides a much higher-level API than the underlying RPC calls that provides simpler language for normal testing tasks on a testnet

Reply:
refactored to high level gecko client